composer.json

```diff
@@ -80,6 +80,7 @@
         "symfony/validator": "4.4.*",
         "symfony/web-link": "4.4.*",
         "symfony/yaml": "4.4.*",
+        "thecodingmachine/safe": "^1.3",
```

Comment:
Note that this was already an implicit dependency, I'm making it explicit as I'm using the `Safe\` methods for robustness.